src/contentChunk.js

```diff
@@ -2,6 +2,16 @@
 import mongodb from 'mongodb'
 import { connectionDefault } from './config'
 
+let bucket
+const getGridFSBucket = () => {
+  if (!bucket) {
+    bucket = new mongodb.GridFSBucket(connectionDefault.client.db())
+    return bucket
```

Comment:
@BMartinos I think that the return here is unnecessary line as it will get triggered outside of the if statement anyway

Reply:
Yeah you are correct. This can be removed :+1